cmd/agent/common/common_nix.go

```diff
@@ -3,8 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-2020 Datadog, Inc.
 
-// +build freebsd netbsd openbsd solaris dragonfly linux
-// +build !android
+// +build netbsd openbsd solaris dragonfly linux
+// +build !freebsd !android
```

Comment:
This means

```
(netbsd OR openbsd OR solaris OR dragonfly OR linux) AND (!freebsd OR !android)
```

which includes this file on Android (since it is `linux` and `!freebsd`) and breaks our build.

I think you can just remove the `!freebsd` tag (reading [the Go docs](https://golang.org/cmd/go/#hdr-Build_constraints) it does not seem to match any other `GOOS` tag as Android does):

```suggestion
// +build !android
```

That would mean:

```
(netbsd OR openbsd OR solaris OR dragonfly OR linux) AND !android
```

which should not match `freebsd`.